web/pdf_page_view.js

```diff
@@ -374,6 +376,10 @@ class PDFPageView {
     });
   }
 
+  hasEditableAnnotations() {
+    return !!this.annotationLayer?.hasEditableAnnotations();
+  }
+
```

Comment:
Nit: This feels like a slightly strange spot for this method, given the private methods it's appearing between.